pkg/controller/seed-controller-manager/preset-controller/doc.go

```diff
@@ -0,0 +1,20 @@
+/*
+Copyright 2021 The Kubermatic Kubernetes Platform contributors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+/*
+Package presetcontroller contains a controller that is responsible for managing presets.
```

Comment:
To be honest, I read the ticket and the PR and I still don't know what this controller does or why. When a preset is deleted, it sets some annotation? Why? What is this used for? Can you please describe the purpose of this controller more than "The x-controller is a controller that controls X." :)